.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: Go
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
```

Comment:
Nix forevah https://github.com/kasuboski/feedreader/blob/main/.github/workflows/ci.yaml#L25

Reply:
I think I set this up correctly.. let me know